lib/medium-admin.php

```diff
@@ -1054,6 +1054,10 @@ private static function _prepare_content($post) {
       $post_content = do_shortcode(wpautop($post->post_content));
     }
 
+		$post_content = apply_filters( 'medium_post_content', $post_content, $post );
```

Comment:
indentation is off and could you please follow the spacing code-style of the rest of the plugin (even though it's not WP-standard).